.github/scripts/lint-ecosystem.js

```diff
@@ -16,26 +16,33 @@ module.exports = async function ({ core }) {
   const moduleNameRegex = /^\- \[\`(.+)\`\]/
   let hasOutOfOrderItem = false
   let lineNumber = 0
-  let inCommmunitySection = false
+  let inCommunitySection = false
   let modules = []
 
   for await (const line of rl) {
     lineNumber += 1
     if (line.startsWith('#### [Community]')) {
-      inCommmunitySection = true
+      inCommunitySection = true
     }
     if (line.startsWith('#### [Community Tools]')) {
-      inCommmunitySection = false
+      inCommunitySection = false
     }
-    if (inCommmunitySection === false) {
+    if (inCommunitySection === false) {
       continue
     }
```

Comment:
I wanted to point out that it currently only accounts for modules in the `#### [Community]`, it doesn't account for either `#### [Community Tools]` nor `#### [Core]`

Reply:
I've re-arranged the order of the conditionals to make the script check for misformatted patterns in all sections, but it only checks for ordering in the community section (the previously existing behavior before I add any changes)

Reply:
I'll keep this comment unresolved, feel free to resolve it if you don't have any comments regarding this